README.md

```diff
@@ -1,5 +1,9 @@
 # PILAR
 
+## Environemnt Setup
+
+Install [poetry](https://python-poetry.org/) and install dependencies using `poetry install`
+
```

Comment:
Let's also add build & run instructions that you included in the README of the PR